tests/integrate/207_NO_KP_OHS_SPIN4/data-1-H.ref

```diff
@@ -1,26 +1,26 @@
- (-0.348876,0) (0,0) (-1.60099e-16,0) (0,0) (-1.34119e-17,0) (0,0) (0.0796737,0) (0,0) (-1.33262e-16,0) (0,0) (1.03717e-16,0) (0,0) (-6.69515e-18,0) (0,0) (-4.17271e-17,0) (0,0) (1.36596e-16,0) (0,0) (-2.17559e-16,0)
- (-0.348876,0) (0,0) (-4.09489e-17,0) (0,0) (-2.22397e-17,0) (0,0) (0.0796737,0) (0,0) (1.0045e-16,0) (0,0) (-2.09114e-17,0) (0,0) (-8.32239e-17,0) (0,0) (7.99308e-16,0) (0,0) (-9.46068e-17,0) (0,0)
- (-0.348876,0) (0,0) (-4.09489e-17,0) (0,0) (-2.22397e-17,0) (0,0) (0.0796737,0) (0,0) (1.0045e-16,0) (0,0) (-2.09114e-17,0) (0,0) (-8.32239e-17,0) (0,0) (7.99308e-16,0) (0,0) (-9.46068e-17,0)
- (0.949625,0) (0,0) (-4.43899e-17,0) (0,0) (-1.13814e-16,0) (0,0) (1.54616e-16,0) (0,0) (3.5284e-16,0) (0,0) (-1.58334e-16,0) (0,0) (-1.98309e-17,0) (0,0) (1.05417e-17,0) (0,0)
- (0.949625,0) (0,0) (-4.43899e-17,0) (0,0) (-1.13814e-16,0) (0,0) (1.54616e-16,0) (0,0) (3.5284e-16,0) (0,0) (-1.58334e-16,0) (0,0) (-1.98309e-17,0) (0,0) (1.05417e-17,0)
- (0.949625,0) (0,0) (-1.72876e-16,0) (0,0) (1.01454e-16,0) (0,0) (-2.25977e-17,0) (0,0) (4.47031e-18,0) (0,0) (-3.39518e-16,0) (0,0) (9.92858e-17,0) (0,0)
- (0.949625,0) (0,0) (-1.72876e-16,0) (0,0) (1.01454e-16,0) (0,0) (-2.25977e-17,0) (0,0) (4.47031e-18,0) (0,0) (-3.39518e-16,0) (0,0) (9.92858e-17,0)
- (0.949625,0) (0,0) (1.19533e-17,0) (0,0) (-7.456e-18,0) (0,0) (-3.13838e-17,0) (0,0) (1.81712e-16,0) (0,0) (1.26337e-16,0) (0,0)
- (0.949625,0) (0,0) (1.19533e-17,0) (0,0) (-7.456e-18,0) (0,0) (-3.13838e-17,0) (0,0) (1.81712e-16,0) (0,0) (1.26337e-16,0)
- (1.69647,0) (0,0) (2.4458e-17,0) (0,0) (2.55026e-18,0) (0,0) (-1.63443e-17,0) (0,0) (3.95666e-17,0) (0,0)
- (1.69647,0) (0,0) (2.4458e-17,0) (0,0) (2.55026e-18,0) (0,0) (-1.63443e-17,0) (0,0) (3.95666e-17,0)
- (1.6968,0) (0,0) (-1.5321e-16,0) (0,0) (3.10594e-18,0) (0,0) (2.105e-17,0) (0,0)
- (1.6968,0) (0,0) (-1.5321e-16,0) (0,0) (3.10594e-18,0) (0,0) (2.105e-17,0)
- (1.6968,0) (0,0) (7.03381e-18,0) (0,0) (-4.86794e-17,0) (0,0)
- (1.6968,0) (0,0) (7.03381e-18,0) (0,0) (-4.86794e-17,0)
- (1.69647,0) (0,0) (-8.34836e-17,0) (0,0)
- (1.69647,0) (0,0) (-8.34836e-17,0)
- (1.6968,0) (0,0)
- (1.6968,0)
+26 (-9.77768381e-01,0.00000000e+00) (0.00000000e+00,0.00000000e+00) (2.60385755e-02,0.00000000e+00) (0.00000000e+00,0.00000000e+00) (-6.66133815e-16,0.00000000e+00) (0.00000000e+00,0.00000000e+00) (2.68163935e-16,0.00000000e+00) (0.00000000e+00,0.00000000e+00) (1.18299642e-16,0.00000000e+00) (0.00000000e+00,0.00000000e+00) (-1.77635684e-15,0.00000000e+00) (0.00000000e+00,0.00000000e+00) (1.97048712e-15,0.00000000e+00) (0.00000000e+00,0.00000000e+00) (2.29525835e-16,0.00000000e+00) (0.00000000e+00,0.00000000e+00) (-3.88578059e-16,0.00000000e+00) (0.00000000e+00,0.00000000e+00) (7.45874074e-17,0.00000000e+00) (0.00000000e+00,0.00000000e+00) (-4.27793809e-17,0.00000000e+00) (0.00000000e+00,0.00000000e+00) (-9.14182776e-16,0.00000000e+00) (0.00000000e+00,0.00000000e+00) (-2.57486475e-18,0.00000000e+00) (0.00000000e+00,0.00000000e+00)
```

Comment:
plz set `out_ndigits` as same as old one, and keep ref.data.

Reply:
thanks, I have added `out_ndigits 5` to all related INPUT files